master/internal/task/tasklogger/logger.go

```diff
@@ -38,6 +40,22 @@ func New(backend Writer) *Logger {
 	return &l
 }
 
+// CreateLogFromMaster creates a tasklog of the format that we expect when it comes from master.
+func CreateLogFromMaster(taskID model.TaskID, level, log string) *model.TaskLog {
```

Comment:
total nit: my dumb ood brain really likes `type T struct {} ... func New() T ... methods for T` to parse "classes". can we move the method out of the middle of Logger